diff -ur zabbix-5.0.0rsm2.1.0alpha1.orig/ui/include/classes/core/CConfigFile.php zabbix-5.0.0rsm2.1.0alpha1/ui/include/classes/core/CConfigFile.php
--- zabbix-5.0.0rsm2.1.0alpha1.orig/ui/include/classes/core/CConfigFile.php	2020-05-15 07:35:29.676128325 +0000
+++ zabbix-5.0.0rsm2.1.0alpha1/ui/include/classes/core/CConfigFile.php	2020-05-15 09:15:10.671167267 +0000
@@ -24,7 +24,7 @@
 	const CONFIG_NOT_FOUND = 1;
 	const CONFIG_ERROR = 2;
 
-	const CONFIG_FILE_PATH = '/conf/zabbix.conf.php';
+	const CONFIG_FILE_PATH = '/etc/zabbix/web/zabbix.conf.php';
 
 	private static $supported_db_types = [
 		ZBX_DB_MYSQL => true,
Only in zabbix-5.0.0rsm2.1.0alpha1/ui/include/classes/core: CConfigFile.php.orig
Only in zabbix-5.0.0rsm2.1.0alpha1/ui/include/classes/core: CConfigFile.php.rej
diff -ur zabbix-5.0.0rsm2.1.0alpha1.orig/ui/include/classes/core/ZBase.php zabbix-5.0.0rsm2.1.0alpha1/ui/include/classes/core/ZBase.php
--- zabbix-5.0.0rsm2.1.0alpha1.orig/ui/include/classes/core/ZBase.php	2020-05-15 07:35:29.676128325 +0000
+++ zabbix-5.0.0rsm2.1.0alpha1/ui/include/classes/core/ZBase.php	2020-05-15 09:21:52.119008753 +0000
@@ -322,7 +322,7 @@
 	 * @throws Exception
 	 */
 	protected function setMaintenanceMode() {
-		require_once 'conf/maintenance.inc.php';
+		require_once '/etc/zabbix/web/maintenance.inc.php';
 
 		if (defined('ZBX_DENY_GUI_ACCESS')) {
 			$user_ip = (isset($_SERVER['HTTP_X_FORWARDED_FOR']) && !empty($_SERVER['HTTP_X_FORWARDED_FOR']))
@@ -338,7 +338,7 @@
 	 * Load zabbix config file.
 	 */
 	protected function loadConfigFile() {
-		$configFile = $this->getRootDir().CConfigFile::CONFIG_FILE_PATH;
+		$configFile = CConfigFile::CONFIG_FILE_PATH;
 		$config = new CConfigFile($configFile);
 		$this->config = $config->load();
 	}
