diff -ur zabbix-5.0.0rsm2.1.0alpha1.orig/ui/include/defines.inc.php zabbix-5.0.0rsm2.1.0alpha1/ui/include/defines.inc.php
--- zabbix-5.0.0rsm2.1.0alpha1.orig/ui/include/defines.inc.php	2020-05-15 09:05:29.986969768 +0000
+++ zabbix-5.0.0rsm2.1.0alpha1/ui/include/defines.inc.php	2020-05-15 09:36:19.641379211 +0000
@@ -87,7 +87,7 @@
 define('ELASTICSEARCH_RESPONSE_DOCUMENTS',		2);
 
 define('ZBX_FONTPATH',				realpath('assets/fonts')); // where to search for font (GD > 2.0.18)
-define('ZBX_GRAPH_FONT_NAME',		'DejaVuSans'); // font file name
+define('ZBX_GRAPH_FONT_NAME',		'graphfont'); // font file name
 define('ZBX_GRAPH_LEGEND_HEIGHT',	120); // when graph height is less then this value, some legend will not show up
 
 define('GRAPH_YAXIS_SIDE_DEFAULT', 0); // 0 - LEFT SIDE, 1 - RIGHT SIDE
@@ -128,7 +128,7 @@
 define('TRIGGERS_OPTION_ALL',				2);
 define('TRIGGERS_OPTION_IN_PROBLEM',		3);
 
-define('ZBX_FONT_NAME', 'DejaVuSans');
+define('ZBX_FONT_NAME', 'graphfont');
 
 define('ZBX_AUTH_INTERNAL',	0);
 define('ZBX_AUTH_LDAP',		1);
